tqdm/_tqdm_custom.py

```diff
@@ -0,0 +1,263 @@
+
+                    bar = '#' * bar_length
+                    frac_bar = chr(48 + frac_bar_length) if frac_bar_length \
+                        else ' '
+
+                # unicode format (if available)
+                else:
+                    bar_length, frac_bar_length = divmod(int(frac * N_BARS * 8), 8)
+
+                    bar = _unich(0x2588) * bar_length
+                    frac_bar = _unich(0x2590 - frac_bar_length) \
+                        if frac_bar_length else ' '
+
+                # whitespace padding
+                if bar_length < N_BARS:
+                    full_bar = bar + frac_bar + \
+                        ' ' * max(N_BARS - bar_length - len(frac_bar), 0)
+                else:
+                    full_bar = bar + \
+                        ' ' * max(N_BARS - bar_length, 0)
+
+                # Piece together the bar parts
+                return l_bar + full_bar + r_bar
+
+            # no total: no progressbar, ETA, just progress stats
+            else:
+                N_BARS = bar_args['n_bars']
+
+                # Custom symbols!
+                # get ascii or unicode template
+                if ascii:
+                    c_symb = bar_format['symbols_indeterminate'].get('ascii', ["====="])
+                    c_symb_rev = bar_format['symbols_indeterminate'].get('ascii_rev', c_symb)
+                else:
+                    c_symb = bar_format['symbols_indeterminate'].get('unicode', ["====="])
+                    c_symb_rev = bar_format['symbols_indeterminate'].get('unicode_rev', c_symb)
+                # looping symbols: just update the symbol animation at each iteration
+                if bar_format['symbols_indeterminate'].get('loop', False):
+                    # increment one step in the animation for each display
+                    self.n_anim += 1
+                    # Get current bar animation based on current iteration
+                    bar = c_symb[divmod(self.n_anim, len(c_symb))[1]]
+
+                    bar_length = N_BARS  # avoid the filling
+                # indeterminate progress bar (cycle from left to right then right to left)
+                else:
+                    # increment one step in the animation for each display
+                    self.n_anim += 1
+                    # Get current bar animation based on current iteration
+                    symbol_idx = divmod(self.n_anim, len(c_symb))[1]
+                    bar = c_symb[symbol_idx]
+                    # Get left filling space and animation step (right pass or left?)
+                    anim_step, fill_left = divmod(self.n_anim, (N_BARS - len(bar)))
+                    # If anim_step is odd, then we do left pass (2nd pass)
+                    if divmod(anim_step, 2)[1] == 1:
+                        # Inverse the left filling space (now it's the right space)
+                        fill_left = N_BARS - len(bar) - fill_left
+                        # Get the reversed symbol
```

Comment:
@lrq3000 🚨 🚨inconsistent comment capitalization

there should be a linter for this.